portal/src/graphql/adminapi/UserDetailsConnectedIdentities.tsx

```diff
@@ -49,39 +52,41 @@ interface UserDetailsConnectedIdentitiesProps {
   availableLoginIdIdentities: string[];
 }
 
-const identityTypes = ["email", "phone", "username"] as const;
-type IdentityType = typeof identityTypes[number];
+const loginIdIdentityTypes = ["email", "phone", "username"] as const;
+type LoginIDIdentityType = typeof loginIdIdentityTypes[number];
+type IdentityType = "login_id" | "oauth";
 
-interface EmailIdentityListItem {
+interface OAuthIdentityListItem {
   id: string;
-  email: string;
+  type: "oauth";
+  providerType: OAuthSSOProviderType;
+  name: string;
   verified: boolean;
   connectedOn: string;
 }
 
-interface PhoneIdentityListItem {
+interface LoginIDIdentityListItem {
   id: string;
-  phone: string;
-  verified: boolean;
-  addedOn: string;
-}
-
-interface UsernameIdentityListItem {
-  id: string;
-  username: string;
+  type: "login_id";
+  key: "email" | "phone" | "username";
+  value: string;
+  verified?: boolean;
```

Comment:
Verified status should not be an optional boolean; instead, it should be a required nullable boolean. If it is null, the login ID identity cannot be verified.

Username login ID can never be verified. Phone & email can verified iff verification is enabled in configuration.

Let's fix that in #385.